ant_plus_sensor/README.md

```diff
@@ -5,16 +5,29 @@
 ## **[Ant-Plus Logger.js](https://github.com/monash-human-power/data-acquisition-system/blob/master/ant_plus_sensor/ant_plus_logger.js)**
 `ant_plus_logger.js` is the only way we can communicate to ant+ devices. It serves as the middle-man between the ANT+ sensors and the DAS MQTT broker. It allows the sensors to show up as a virtual wireless sensor module.
 
-## **Getting Started**
+## Getting Started
 
 ### Setting up node.js libraries
 To install the necessary libraries for node.js you may need to manually install them instead of running `npm install`.
 
 Run the script with `node ant_plus_logger.js`.
 
-To start this script automatically, see [the services README.md](/services/README.md).
+Sometimes, root permissions are required to connect to the USB dongle, so `sudo` may be required.
 
-## **Legacy Code**
+## Deployment
+
+To start this script automatically, follow the instructions in [the services README.md](/services/README.md).
+
+You will also need to ensure the default `pi` user has permissions to access the ANT+ dongle without `sudo`. Create the following file in `/etc/udev/rules.d/50-antplus.rules` and reboot:
+
+```
+# Allow access to ANT+ USB without sudo
+# https://www.xmodulo.com/change-usb-device-permission-linux.html
+SUBSYSTEMS=="usb", ATTRS{idVendor}=="0fcf", ATTRS{idProduct}=="1008", GROUP="users", MODE="0666"
+SUBSYSTEMS=="usb", ATTRS{idVendor}=="0fcf", ATTRS{idProduct}=="1009", GROUP="users", MODE="0666"
```

Comment:
Would any of this need to change if we buy a new Ant plus dongle? Just wondering if any of these details are specific to the dongle or common across them all.

Reply:
Nope, not unless a new version is released. `0x1008` is for version 2 of the garmin ant+ dongle, and `0x1009` for version 3. Right now they (and clones) are the only two dongles supported by the ant plus library.